testing/howard-bc/APKBUILD

```diff
@@ -32,11 +32,13 @@ doc() {
 	# Doc files
 	install -Dm644 "$builddir"/LICENSE.md \
 			"$subpkgdir"/usr/share/doc/$pkgname/LICENSE.md
+
+	rm -rf "$pkgdir"/usr/share/man/
```

Comment:
why it removes? I guess  packaging should do the trick

Reply:
This was the line that removed a warning. The warning said

```
>>> WARNING: howard-bc*: Found /usr/share/man but package name doesn't end with -doc
```

That seemed the right thing to do since it already has a `*-doc` package, but you know better.

Reply:
Is this removing empty directories or are some files not being moved to the `-doc` package?

Reply:
I think it is an empty directory.